subjects/changes/README.md

```diff
@@ -2,16 +2,11 @@
 
 ### Instructions
 
-Imagine you are working on a software to control smart lights in a house. You have access to an array of all the lights in a house.
+Imagine you are working on some software to control smart lights in a house. You have access to an array of all the lights in that house.
 
-Define the associated **function** `new` to the data structure `Light` which creates a new light with the alias passed in the arguments and a brightness of 0.
+Define the associated **function** `new`, and add it to the data structure `Light`. It should create a new light with the alias passed as an argument, with a brightness of .
```

Comment:
its missing the 0 at the end " brightness of 0"

Reply:
Done